### PR TITLE
Whats new blurbs

### PR DESCRIPTION
**Pull request by taldcroft**
### Description

I started writing what's new blurbs for PR's in my packages but then kept going and finished (I think?) all the remaining PR's for 4.3 with a what's new label.

I don't feel strongly about what I wrote in case other maintainers want different text, but at least we have something now before the feature freeze.

@mhvk @pllim @saimn @matteobachetti

### CODE REVIEW

File: docs/whatsnew/4.3.rst [hunk 1]
```diff
@@ -81,6 +85,82 @@ instance on a cluster) by setting ``iers.conf.auto_download = False``. This can
 be done after importing astrpoy but prior to performing any ``Time`` scale
 transformations related to UTC.
 
+.. _whatsnew-4.3-ecsv-subtypes:
```
Comment (saimn) [inline docs/whatsnew/4.3.rst:88]: Can you fix the "astrpoy" 2 lines above ? ;)

File: docs/whatsnew/4.3.rst [hunk 2]
```diff
@@ -81,6 +85,82 @@ instance on a cluster) by setting ``iers.conf.auto_download = False``. This can
...
+QDP (`Quick and Dandy Plotter <https://wwwastro.msfc.nasa.gov/qdp/>`_) format.
+This specialized format is used by some missions such as `Swift
+<https://www.nasa.gov/mission_pages/swift/main>`_.
+
+.. _whatsnew-4.3-fits-append-table:
+
+Append table to existing FITS file
+==================================
+
+It is now easy to append a table to an existing FITS file using a new ``append``
```
Comment (saimn) [inline docs/whatsnew/4.3.rst:125]:
```suggestion
It is now easy to append a :ref:`Table <astropy-table>` to an existing FITS file using a new ``append``
```
Adding ref so there is no confusion with FITS tables.

**Review by saimn — APPROVED** [review]
Thanks for doing it, seems really good, and a nice list of new features !

**Comment by pllim** [comment]
I replaced myself with @eteq as he is handling the actual release this time.

File: docs/whatsnew/4.3.rst [hunk 3]
```diff
@@ -81,6 +85,82 @@ instance on a cluster) by setting ``iers.conf.auto_download = False``. This can
...
+.. _whatsnew-4.3-ecsv-subtypes:
+
+Support for multidimensional and object columns in ECSV
+=======================================================
+
+Support has been added for reading and writing Tables with three additional data
+subtypes:
+
+- Multidimensional column data (both masked and unmasked) with fixed dimensions
+  in all table cells
```
Comment (mhvk) [inline docs/whatsnew/4.3.rst:97]: Missing period at end of line

**Review by mhvk — APPROVED** [review]
Thanks!  Masked class text looks good. I only managed to find a missing period - but perhaps just didn't look carefully enough!

**Comment by taldcroft** [comment]
Thanks for the quick review. I made the suggested changes and will just wait for the RTD build.

**Pull request merged**